solidity/scripts/liquidity-rewards-starter.js

```diff
@@ -39,105 +41,72 @@ const mintAndApproveLPReward = async (
   })
 }
 
+const getWrappedTokenContract = async (LPRewardsContract) => {
```

Comment:
We're expecting a contract instance here so maybe name it starting with lowercase as we do elsewhere?
```suggestion
const getWrappedTokenContract = async (lpRewardsContract) => {
```